samples/bluetooth/iso_connected_benchmark/src/main.c

```diff
@@ -30,7 +30,7 @@ enum benchmark_role {
 
 #define DEFAULT_CIS_RTN         2
 #define DEFAULT_CIS_INTERVAL_US 7500
```

Comment:
This `DEFAULT_CIS_INTERVAL_US`  is actually SDU interval, right?

Reply:
Yes, you are right. See `hci_le_set_cig_params` in `iso.c`